Add detailed instructions for installing prerequisites on Windows (#33)
brown-ccv/honeycomb-docs: docs/firebase.md

- Create a firebase project by clicking add project and enter a Project Name.

### 2. Linking firebase to task

- Login to firebase using `firebase login` command in the terminal.
- Navigate to the `.firebaserc` file home directory and edit the `"default"` field with the project name given in part 1.

```json
{
    "projects": {
        "default": "<your project name>"
    }
}
```

### 3. Copying web app credentials

- Navigate to the [firebase console](https://console.firebase.google.com/) and select the project.
- Create a new Web App by clicking on `Add App` or the `</>` code symbol and following the prompts.
- Enter the a name for the Firebase app (could be the same as your Honeycomb task repo name).
- Check "Also set up Firebase Hosting for this app."
- Click `Register App`. This should auto-generate a script with several values that you need to copy into the next step.
- Create a new file, `.env.firebase` in the `env/` folder and copy the following lines:

```text
REACT_APP_FIREBASE="true"
REACT_APP_apiKey=
REACT_APP_authDomain=
REACT_APP_projectId=
REACT_APP_storageBucket=
REACT_APP_messagingSenderId=
REACT_APP_appId=
```

> - Copy the auto-generated values from the Firebase console to the corresponding variables in the `.env.firebase` you just created

Reviewer: ```suggestion
- Copy the auto-generated values from the Firebase console to the corresponding variables in the `.env.firebase` you just created. You only need to copy the variables referenced here. If you enabled Google Analytics during project creation, additional variables may be printed in the console. You don't need to worry about copying those.
```

Honeycomb comes with methods and configurations for easy [Firebase](https://firebase.google.com/) and [Cloud Firestore](https://firebase.google.com/docs/firestore) set up. Please use this section to configure [Firebase Hosting](https://firebase.google.com/docs/hosting) services and firestore cloud storage.

## Setting up Firebase

### 0. Installing the Firebase Command Line Interface

> If the firebase cli is not installed on your computer it's best to install it now. To check if it is install simply type `firebase` into a terminal window.

Reviewer 2: Reducing text

```suggestion
To check if firebase has been installed, type `firebase` into a terminal window.
```

```terminal
firebase
```

If a list of options are printed on the terminal then the firebase tool has already been installed. If not, you will get an error - something along the lines of "Command not found." The firebase-cli can then be installed using npm:

```terminal
npm install -g firebase-tools
```

_The `-g` flag installs an npm package globally on your system, as opposed to the current project_

### 1. Initializing firebase account and project

- Create and login to a firebase account on [the Firebase website](https://firebase.google.com/).
- Create a firebase project by clicking add project and enter a Project Name.

### 2. Linking firebase to task

> - Install [Firebase CLI](https://firebase.google.com/docs/cli) on your local computer.

Reviewer 2: Wasn't this done in the previous step 0?

Author: Good catch!
